stable/kibana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: kibana
-version: 2.0.0
-appVersion: 6.6.1
+version: 3.0.0
```

Comment:
is this a breaking change for people coming from <= 2.0? 
if not and its only a new feature chart version 2.1.0 would be better.

Reply:
set `2.1.0`